pynamodb/attributes.py

```diff
@@ -772,7 +772,36 @@ def deserialize(self, value):
 
 class VersionAttribute(NumberAttribute):
     """
-    A version attribute
+    A number attribute that denotes a version / revision counter of the model.
```

Comment:
TODO simply say it's an optimistic locking mechanism and move the prose to a separate page